TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/BasicMotorTest.java

```diff
@@ -10,35 +10,44 @@
 /**
  * TeleOp OpMode for testing motors.
  */
+// "BasicMotorTest" is the name of OpMode in Driver Station
 @TeleOp(name = "BasicMotorTest", group = "Iterative OpMode")
+
 public class BasicMotorTest extends OpMode {
     private final ElapsedTime time = new ElapsedTime();
-    HWC robot;
+    HWC robot; // Declares roboBt as an instance of a hardware class.
 
+    // Runs once when "INIT" button is pressed.
     @Override
     public void init() {
         telemetry.addData("Status", "Initializing");
-        robot = new HWC(hardwareMap, telemetry);
-        telemetry.addData("Status", "Initialized");
+        // Telemetry.addData stores data in a hashmap-like data set.
+        robot = new HWC(hardwareMap, telemetry); // assigns robot value
+        telemetry.addData("Status", "Initialized"); // Updates "Status" key's value.
     }
 
+    // Runs repetitively between "INIT" and "START" button presses,
+    // FRC SDK default version does nothing.
     @Override
     public void init_loop() {
     }
 
+    // Runs once after "START" button pressed.
     @Override
     public void start() {
         time.reset();
     }
 
+    // Runs forever after "START" button pressed
+    // (unless given an exception or "STOP" button pressed).
     @Override
     public void loop() {
-        if (gamepad1.a) {
-            robot.leftFront.setPower(1.0);
+        if (gamepad1.a) { // press button 'A'
```

Comment:
Don't put your comments on the same line here. This is also a redundant comment.